docs/kyma-environment-broker/02-03-cls-integration.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Cloud Logging Service Integration
+type: Architecture
+---
+
+The Cloud Logging service (CLS) is a managed logging service for shipping logs from various runtimes such as SKR. It uses the Elastic Stack (open source logging platform Elasticsearch, Fluentd, Kibana) to store, parse, and visualize the application log data coming from Kubernetes, Cloud Foundry applications or any other sources.
+
+ Kyma Environment Broker (KEB) provisions a CLS instance for a global account and creates a binding per SKR cluster. It passes the CLS credentials to the provisioner as SKR overrides, so that Fluent Bit in the SKR is properly configured to push the logs to the CLS instance. The following architecture diagram explains the complete process:
+
+![CLS diagram](./assets/cls-arch.svg)
+
+1. CSI sends the request to KEB to provision a new SKR.
+  - If there is no CLS instance present yet for the global account, KEB provisions a new instance of CLS for the global account.
+
+      - If the CLS instance is already provisioned for the global account, then KEB creates a binding. see [3.](3)
```

Comment:
With rephrased step 2, no need for this second bullet anymore. Binding creation is described in step 3.

```suggestion
```